pyad/test_reverse.py

```diff
@@ -1,20 +0,0 @@
-from reverse_mode import *
```

Comment:
This file ran in python with no problem. But Travis said "ModuleNotFoundError: No module named 'reverse_mode'" for some reason. Any ideas?

Reply:
I think it's because in Travis, it goes by the path from the root of the repository. So, you would need to do `from pyad.reverse_mode import *`.

But, I think it might be better if you wrote `import pyad.reverse_mode as rev` and used `rev.Tensor()`, `rev.sin(x)`, etc

It looks like you deleted this file and moved the code to the `reverse_mode.py` file. I think you should keep this file. But make sure it's in the `tests/` directory and within a `test_xxx()` function like we have in `test_pyad.py`. We'll probably rename `test_pyad.py` to be `test_forward.py`